### ISSUE TITLE
Add development instructions

### ISSUE THREAD

**Issue by ralphbean** [fedora-infra/datanommer#3]
Add a set of instructions to the README for how to hack on datanommer.

Right now, the only instructions are how to install it from yum.  It would be useful for people who want to develop on it to know how to set it up from git.

**Comment by ralphbean** [comment]
These instructions will need to include a walkthrough of how to set up and use the `virtualenv` and/or `virtualenvwrapper` tool.  You can mimic or mirror the walkthrough for fedmsg instructions found here http://fedmsg.readthedocs.org/en/latest/development/

Note that, this is a little more tricky for datanommer since there are three namespace packages you'll have to run `python setup.py develop` for:  `datanommer.models`, `datanommer.commands`, and `datanommer.consumer`.  And order matters here..  If you run it on `datanommer.commands` first, then it will pull in the `datanommer.models` from PyPI instead of your local development copy.

**Comment by ralphbean** [comment]
@sontek made some notes - http://paste2.org/p/2526070

**Comment by sontek** [comment]
You can also save this as setup.sh:

```
#/usr/bin/env bash

for fname in "datanommer.models" "datanommer.commands" "datanommer.consumer"
do
    cd "$fname"
    python setup.py develop
    cd ..
done

python setup.py develop
```

then run:
chmod +x setup.sh
./setup.sh

in the root of the datanommer folder and it'll get you going in the correct order

**Comment by ralphbean** [comment]
@silverrain took this on in 56ff01648120d852ea7fe60a3095ef15d8aefe62 and 9ce9ceff191cc161e9f96f4da5959de23b9be447.

Nice work.  @silverrain, do you want to close the issue?

**Comment by sontek** [comment]
Nice! Looks good! I made a few comments on the diff for things that I think could improve it.

**Comment by ralphbean** [comment]
That discussion in the diff can be found here in #11.

**Comment by ralphbean** [comment]
Weird.  It doesn't seem to be working.  I receive an email, but the
issue still appears to be open.

**Comment by silverrain** [comment]
Closed #3.
